code/modules/mob/dead/observer/observer_base.dm

```diff
@@ -35,6 +35,7 @@ GLOBAL_VAR_INIT(observer_default_invisibility, INVISIBILITY_OBSERVER)
 	///toggle for ghost gas analyzer
 	var/gas_analyzer = FALSE
 	var/datum/orbit_menu/orbit_menu
+	var/datum/ui_module/crew_monitor/ghost/crew_monitor
```

Comment:
I wonder if it would be better to make this global and have a single one for all observers, since I think it uses only `useLocalState`

Reply:
Good question. I don't know if there are any other UIs that work like that, I mostly just imitated what we had for the orbit menu

Reply:
I'd recommend something like `GLOBAL_DATUM_INIT(generic_crew_manifest, /datum/ui_module/generic_crew_manifest, new)` or `GLOBAL_DATUM_INIT(ghost_hud_panel, /datum/ui_module/ghost_hud_panel, new)`

Reply:
Implemented, seems like this works pretty well. Thanks for the suggestion!